cmd/whereabouts.go

```diff
@@ -18,18 +19,24 @@ import (
 )
 
 func main() {
+	var startTime time.Time
 	skel.PluginMain(func(args *skel.CmdArgs) error {
 		ipamConf, confVersion, err := config.LoadIPAMConfig(args.StdinData, args.Args)
 		if err != nil {
 			logging.Errorf("IPAM configuration load failed: %s", err)
 			return err
 		}
-		logging.Debugf("ADD - IPAM configuration successfully read: %+v", *ipamConf)
+
+		startTime = time.Now()
+		logging.Debugf("ADD - IPAM plugin start. Config: %+v", *ipamConf)
 		ipam, err := kubernetes.NewKubernetesIPAM(args.ContainerID, *ipamConf)
 		if err != nil {
 			return logging.Errorf("failed to create Kubernetes IPAM manager: %v", err)
 		}
-		defer func() { safeCloseKubernetesBackendConnection(ipam) }()
+		defer func() {
+			safeCloseKubernetesBackendConnection(ipam)
+			logging.Debugf("ADD - IPAM plugin finished. Took: %q Config: %+v", time.Since(startTime).String(), *ipamConf)
+		}()
```

Comment:
Please consider using a function that gets a string, startTime and is reused in both places
as beside ADD / DEL string it is the same